SETUP/upgrade/15/20201216_alter_projects.php

```diff
@@ -0,0 +1,23 @@
+<?php
+$relPath='../../../pinc/';
+include_once($relPath.'base.inc');
+
+header('Content-type: text/plain');
+
+// ------------------------------------------------------------
+
+echo "Adding comment_format  column to projects table..\n";
+$sql = "
+    ALTER TABLE projects
+        ADD COLUMN comment_format varchar(8) NOT NULL DEFAULT 'html' AFTER comments
```

Comment:
This default column value here and the one in `db_schema.sql` should match, and I think `markdown` is the correct one. So we should also add an update query in this file to set all existing projects to `html`.

Reply:
ah yes, I confused myself with default value for these projects and default constraint. I tried to google around for a way to alter a table by adding a column with default constraint with different value, but 2 statements should be fine.